test/edu-benefits/1990e/schema/maximal-test.json

```diff
@@ -34,9 +34,11 @@
         "last": "Lkjh",
         "suffix": "Jr."
       },
-      "veteranSocialSecurityNumber": "556789999",
-      "view:noSSN": true,
-      "vaFileNumber": "C12345678",
+      "view:veteranId": {
```

Comment:
Why was this not here before? Was it just copied from somewhere else, or did it get flattened?